src/modules/dashboard/reducers/allDomains.ts

```diff
@@ -43,6 +43,10 @@ const allDomainsReducer: ReducerType<AllDomainsMap> = (
         meta: { key },
         payload: { domains },
       } = action;
+      /* Add root domain to response which is the parent of all domains  */
```

Comment:
I think this should be added to the query, because that's the source of truth, and it should make things clearer. The root domain object could be added to the start of this array: https://github.com/JoinColony/colonyDapp/blob/175e6f72bc6875551c00d8f74ec856343e70a52b/src/modules/dashboard/data/queries/colonies.ts#L468

Reply:
@JamesLefrere Just moved the change to the query! :white_check_mark: